app/lib/events/web_request.rb

```diff
@@ -13,19 +13,17 @@ def as_json
     end
 
     def with_request_details(rack_request)
-      @event_hash.merge!(
-        request_uuid: rack_request.uuid,
-        request_data: {
-          path: rack_request.path,
-          method: rack_request.method,
-          user_agent: rack_request.user_agent,
-        }
-      )
+      @event_hash[:request_uuid] = rack_request.uuid
```

Comment:
Out of curiosity, why this change of style? I think I find a hash literal easier to read

Reply:
The `.merge!` were overwriting hash contents and I decided it wasn't worth expending my (very limited) brain power on sorting it out. 😄 Happy to take a suggest code change on this.

Reply:
Figured out the Right :tm: way to do this. Thanks for the prodding.